### PR TITLE
[About] Move contributors to CONTRIBUTORS.md and add current people to the Thank You list

### PR DESCRIPTION
**Pull request by andygrunwald**
### What is this PR doing?

[About] Move contributors to CONTRIBUTORS.md and add current people to the Thank You list

### Which issue(s) this PR fixes:

Related #409

### Checklist

- [X] `CHANGELOG.md` entry
- [X] Documentation update

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Thanks for your effort
+
+Thanks goes to all contributors to this project.
+
+Have a look at https://github.com/lansuite/lansuite/graphs/contributors to see all contributors.
```
Comment (M4LuZ) [inline CONTRIBUTORS.md:5]:
```suggestion
Have a look at https://github.com/lansuite/lansuite/graphs/contributors to see current contributors.
```
As listing does not show unmapped SVN accounts

File: modules/about/credits.php [hunk 2]
```diff
@@ -4,26 +4,8 @@
...
-$dsp->AddSingleRow('<b>Erweiterungen, Optimierungen, Fehlerbeseitigung</b>', 'align="center"');
-$dsp->AddSingleRow('Raphael Mansmann<br />Jochen Jung<br />Marco Müller', 'align="center"');
-
-$dsp->AddSingleRow('<b>Modul-Programming</b>', 'align="center"');
-$dsp->AddSingleRow('Wir danken allen Modulprogramierern.</b><br /> Im Modulkonfigurator kann man nachsehen wer welches Modul geschrieben hat', 'align="center"');
-
-$dsp->AddSingleRow('<b>On-line Documentation</b>', 'align="center"');
-$dsp->AddSingleRow('Sebastian Haslinger', 'align="center"');
+$dsp->AddSingleRow('<b>Thanks to all project contributos</b>', 'align="center"');
+$dsp->AddSingleRow('Please head over to <a href="https://github.com/lansuite/lansuite/blob/master/CONTRIBUTORS.md" title="All lansuite contributors">project contributors</a> to see the full list of project contributors.', 'align="center"');
```
Comment (M4LuZ) [inline modules/about/credits.php:8]: What do you think about including the page in the webpages generated by docusaurus?
Also same as above: Git AFAIK lists current contributors, not a full list of all people ever contributing

**Review by M4LuZ — APPROVED** [review]
I'm generally fine with this, see two comments before merging.

Scratching my head a bit why this was considered to be necessary now, though ;)

**Comment by andygrunwald** [comment]
> Scratching my head a bit why this was considered to be necessary now, though ;)

It was not fully necessary, still improves the product in a little bit.

File: modules/about/credits.php (hunk 2 above)
Comment (andygrunwald) [reply modules/about/credits.php:8]: Not all contributors are listed, this is true. I think we will never find all contributors again. What we have now is all what we got so far.

Not sure if we need to add this to the documentation. I think mostly that this is information from/for development.

**Pull request merged**